scripts/shared.sh

```diff
@@ -25,7 +25,7 @@ function get_short_version() {
 }
 
 function get_aws_account() {
-    aws --region us-west-2 sts --endpoint-url https://sts.us-west-2.amazonaws.com get-caller-identity --query 'Account' --output text
+    aws --region $AWS_DEFAULT_REGION sts --endpoint-url https://sts.$AWS_DEFAULT_REGION.amazonaws.com get-caller-identity --query 'Account' --output text
```

Comment:
this change appears unrelated. why is it needed? and is this env var alway populated? the get_default_region function at L11 implies that it is not.

Reply:
It's a correction to the previous commit. I changed it to us-west-2 to run tests on my end and didn't change it back in previous commits. It should be $AWS_DEFAULT_REGION (https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/scripts/shared.sh#L28), we've been using this since this commit: https://github.com/aws/sagemaker-tensorflow-serving-container/commit/777cccc71de90029b9e59f4a204dfc47af9f68df

Reply:
ok